containers/docker-compose.yaml

```diff
@@ -137,6 +137,9 @@ services:
       - GDS_BFF_DATABASE_INSECURE=true
       - GDS_BFF_DATABASE_CERT_PATH
       - GDS_BFF_DATABASE_POOL_PATH
+      - GDS_BFF_SENTRY_TRACK_PERFORMANCE
+      - SENTRY_DSN
+      - SENTRY_ENVIRONMENT
```

Comment:
Would you mind adding all of these environment variables to `.env.template` as well?